app/controllers/sections/info/DestinationTypeController.scala

```diff
@@ -52,33 +52,32 @@ class DestinationTypeController @Inject()(
                                            val userAllowList: UserAllowListAction
                                          ) extends BasePreDraftNavigationController with AuthActionHelper with PreDraftAuthActionHelper {
 
-  def onPreDraftPageLoad(ern: String, arc: String, mode: Mode): Action[AnyContent] =
+  def onPreDraftPageLoad(ern: String, arc: String): Action[AnyContent] =
     authorisedWithPreDraftDataUpToDateMovementAsync(ern, arc) { implicit request =>
-      renderView(Ok, fillForm(DestinationTypePage, formProvider()), mode)
+      renderView(Ok, fillForm(DestinationTypePage, formProvider()))
```

Comment:
I think that the DestinationTypePage should now be NewDestinationTypePage?

Reply:
It currently looks like this in Mongo which could be misleading:
![image](https://github.com/hmrc/emcs-tfe-change-destination-frontend/assets/53828896/8c97506d-7d21-4c1f-b7a0-a40cefbe2f28)